features/closewatcher.yml

```
@@ -0,0 +1,11 @@
+name: CloseWatcher
+description: "The CloseWatcher API listens and responds to close requests for developer-implemented user interface components such as sidebars or custom keyboards."
+spec: https://html.spec.whatwg.org/multipage/interaction.html#closewatcher
```

I think this link gives better context:

```suggestion
spec: https://html.spec.whatwg.org/multipage/interaction.html#close-requests-and-close-watchers
```